src/libraries/System.Memory/ref/System.Memory.cs

```diff
@@ -219,6 +219,7 @@ public abstract partial class ReadOnlySequenceSegment<T>
         public long Length { get { throw null; } }
         public System.SequencePosition Start { get { throw null; } }
         public System.Buffers.ReadOnlySequence<T>.Enumerator GetEnumerator() { throw null; }
+        public long GetOffset(SequencePosition position) { throw null; }
```

Comment:
I did it, but I need to only move new api because gen tool update a lot of signature, I think needs some update 

```
D:\git\runtime\src\libraries\System.Memory\ref (getoffset -> origin)
λ  dotnet msbuild /t:GenerateReferenceSource /p:TargetFramework=netcoreapp5.0-Windows_NT
```

![image](https://user-images.githubusercontent.com/7894084/74333053-706fd880-4d97-11ea-8ca4-c67bf24b80bc.png)

Reply:
@safern, @Anipik  - do you know where we are with supporting unmanaged/readonly properties/etc. in GenAPI so that regeneration isn't this noisy?